pkg/operator/k8sutil/pod.go

```diff
@@ -297,8 +297,9 @@ func deleteResourceAndWait(namespace, name, resourceType string,
 }
 
 // Environment variables used by storage cluster daemons
-func ClusterDaemonEnvVars() []v1.EnvVar {
+func ClusterDaemonEnvVars(image string) []v1.EnvVar {
```

Comment:
Nit here, `image` could be the Rook or Ceph image, so I think it would be best to explicitly name this `cephImage`.

Reply:
it's already been merged, so we could change it in a future PR